src/game/client.cpp

```diff
@@ -3240,142 +3240,29 @@ namespace client
         }
     }
 
-    int serverstat(serverinfo *a)
+    bool serverinfocompare(serverinfo const*a, serverinfo const*b)
     {
-        if(a->attr.length() > 4 && a->numplayers >= a->attr[4])
+        int comp = a->version_compare( *b );
+        if( comp != 0 )
```

Comment:
The motivation behind these calls is not clear. There should be additional documentary comments in the code.